<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwidget</class>
 <widget class="QWidget" name="mainwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainwidget</string>
  </property>
  <widget class="TitleWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>680</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>680</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="ChannelWidget" name="channelWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>680</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>680</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>680</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="ControlPanel" name="controlWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>680</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>680</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>680</width>
     <height>250</height>
    </size>
   </property>
  </widget>
  <widget class="LoginPanel" name="loginWidget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>500</width>
     <height>0</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header>titlewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChannelWidget</class>
   <extends>QWidget</extends>
   <header>channelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>controlpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoginPanel</class>
   <extends>QWidget</extends>
   <header>loginpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
